cli/image.go

```diff
@@ -67,7 +67,7 @@ func importImage(clusterName string, images []string, noRemove bool) error {
 		if err = docker.ContainerRemove(ctx, toolsContainerID, types.ContainerRemoveOptions{
 			Force: true,
 		}); err != nil {
-			log.Println(fmt.Errorf("WARN: couldn't remove tools container\n%+v", err))
+			log.Warning(fmt.Errorf("Couldn't remove tools container\n%+v", err))
```

Comment:
Could as well be `log.Warningf` instead of using `fmt.Errorf` inside

Reply:
Done.

Reply:
It has to be `log.Warningf("Couldn't remove tools container\n%+v", err)` with the trailing `f`, because the message includes a formatting directive :+1:

Reply:
Done.